x-pack/plugins/apm/public/components/alerting/helper.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import datemath from '@elastic/datemath';
+
+export function windowToTimeRange(windowSize: number, windowUnit: string) {
```

Comment:
`windowToTimeRange` seems a little vague. WDYT about `getAbsoluteTimeRange`?

```suggestion
export function getAbsoluteTimeRange(windowSize: number, windowUnit: string) {
```